src/test/java/qna/domain/QuestionTest.java

```diff
@@ -1,6 +1,77 @@
 package qna.domain;
 
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.List;
+import java.util.Optional;
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+@DataJpaTest
 public class QuestionTest {
     public static final Question Q1 = new Question("title1", "contents1").writeBy(UserTest.JAVAJIGI);
     public static final Question Q2 = new Question("title2", "contents2").writeBy(UserTest.SANJIGI);
+
+    @PersistenceContext
+    private EntityManager em;
+
+    @Autowired
+    private QuestionRepository questionRepository;
+
+    @Test
+    @DisplayName("Question 도메인 생성 테스트")
+    void generate01(){
```

Comment:
```suggestion
    void generate01() {
```
@_@ `IDE`에 있는 자동 포멧팅을 적용해보아요~ :)